protos/transaction.proto

```diff
@@ -154,6 +154,17 @@ message Transaction {
     uint32 num_fragments = 1;
   }
 
+  // An operation that updates rows but does not add or remove rows.
```

Comment:
We can tackle this in the merge PR but I wasn't sure if we wanted merge to be a separate operation or just reuse update (I'm reusing update at the moment).